openpype/hosts/maya/plugins/publish/extract_maya_scene_raw.py

```diff
@@ -23,6 +25,16 @@ class ExtractMayaSceneRaw(publish.Extractor):
                 "camerarig"]
     scene_type = "ma"
 
+    @classmethod
+    def get_attribute_defs(cls):
+        return [
+            BoolDef(
+                "preserve_references",
+                label="Preserve References",
```

Comment:
Would this benefit from a tooltip?
```suggestion
                label="Preserve References",
                tooltip=(
                    "When enabled references will still be references "
                    "in the published file.\nWhen disabled the references "
                    "are imported into the published file generating a "
                    "file without references."
                ),
```